fitbenchmarking/fitbenchmarking
# Add support for numdifftools

#### Description of Work

Fixes #571
Fixes #703 

#### Testing Instructions

1. install fitbenchmarking with numdifftools included, check all works
2. verify the numdifftools jacobian works on a range of examples and minimizers

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
